Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/suite_view.py

```diff
@@ -58,7 +58,11 @@ def _suites_for_configuration(
         AssertRequest.query_kwargs_empty(**kwargs)
 
         with self.upload_controller.upload_context:
-            suites_by_configuration = self.upload_controller.upload_context.find_suites(configurations=configurations, recent=boolean_query(*recent)[0] if recent else True)
+            suites_by_configuration = self.upload_controller.upload_context.find_suites(
+                configurations=configurations,
+                recent=boolean_query(*recent)[0] if recent else True,
+                branch=branch[0] if branch else None,
```

Comment:
Can we always assume branch is a list if it's not `None`?

Reply:
Tuple, actually, but yes! `branch` comes from `query_as_kwargs `, which comes from https://github.com/WebKit/WebKit/blob/474074a02f3ae58188593ed1e15cfcd2b6855d1e/Tools/Scripts/libraries/webkitflaskpy/webkitflaskpy/util.py#L64. That function parses incoming URL query parameters and puts them into a list. From an API perspective, the `branch` query parameter is only supposed to have a single argument, but we don't actually raise an exception when that assumption is violated, we just ignore everything except the first one.